internal/pkg/tfconf/module.go

```diff
@@ -129,6 +152,7 @@ func CreateModule(path string) (*Module, error) {
 		outputs = append(outputs, &Output{
 			Name:        output.Name,
 			Description: String(outputDescription),
+			Value:       terraformOutputs[output.Name].Value,
```

Comment:
I just realized that if a user passes the `inject-output-value` flag with a file which is missing an output, they will run into errors. I need to fix this.

Reply:
To fix that, you can change for this:
```
output := &Output{...}
if ok, value := terraformOutputs[output.Name]; ok {
  output.Value = value
}
outputs = append(outputs, output)
```